src/Chainweb/Pact/PactService.hs

```diff
@@ -34,6 +34,7 @@ module Chainweb.Pact.PactService
     , initPactService'
     , execNewGenesisBlock
     , getGasModel
+    , chainweb213GasModel
```

Comment:
Why does the simulator need this gas model? Is it special like "the latest gas model" or something? If so, would it make sense to have an alias for it and export that?

Reply:
It would require functionality to actually compute the gas model at a given height, so I opted with the least complicated approach which is to use the gas model we're using currently and address when that's no longer the case (which is probably soon no?)

Reply:
Actually upon review there's `getGasModel` LOL so I'll use that